Dockerfile.erb

```diff
@@ -1,20 +1,10 @@
-FROM debian:jessie
+FROM debian:stable
```

Comment:
I think it's not a good idea to pin to `debian:stable` here, because when the stable tag changes (debian 10 or whatever) it might break things.

Instead, we should pin to `debian:stretch`. I did some testing and it looks like `debian:stretch-slim` will actually work fine for us which is 45% smaller again.